spec/models/database_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe "Database" do
+  excluded_models = ["Gateway", "PayPalExpress", "Bogus", "BogusSimple"].freeze
+
+  it "should have foreign keys for models with a belongs_to relationship" do
+    pending "Consider adding foreign keys"
```

Comment:
Instead of pending, you could create an array of models or associations which still need doing, like the Rubocop todo list.